nautilus_trader/common/component.pxd

```diff
@@ -288,7 +288,7 @@ cdef class MessageBus:
     cpdef void add_streaming_type(self, type cls)
     cpdef void send(self, str endpoint, msg)
     cpdef void request(self, str endpoint, Request request)
-    cpdef void response(self, Response response)
+    cpdef void response(self, Response response, bint remove_callback=*)
```

Comment:
Do you think the bar aggregation machinery is encroaching too much into response handling and correlation IDs here? 

I think this is only necessary because of the recursion which is being used later on. (are we also able to avoid recursion?)

Is there another way this can be handled by adding another class member?

Reply:
Resolved, concatenating different aggregated bar types to a single list response instead.